benchmarks/src/main.cpp

```diff
@@ -0,0 +1,20 @@
+#include "./construction_benchmarks.hpp"
+#include "./unit_benchmarks.hpp"
+
+auto main() -> int
+{
+    run_default_onto_reserved_vector_benchmarks();
+    run_rand_onto_reserved_vector_benchmarks();
+    run_copy_vector_of_benchmarks();
+
+    run_add_units_benchmarks();
+    run_add_units_simd_benchmarks();
+    run_subtract_units_benchmarks();
+    run_subtract_units_simd_benchmarks();
+    run_multiply_units_benchmarks();
+    run_multiply_units_simd_benchmarks();
+    run_divide_units_benchmarks();
+    run_divide_units_simd_benchmarks();
+
+    return 0;
+}
```

Comment:
These examples suggest to use doctest as a (benchmark/test) runner, so you don't have to keep track of them at different levels.

Reply:
And if you read at the bottom of the previous section, there is a text describing that is how the author implements benchmarks:

> Nanobench does not come with a test runner, so you can easily use it with any framework you like. In the remaining examples, I’m using [doctest](https://github.com/onqtam/doctest) as a unit test framework.